modules/core/shared/src/main/scala/lucuma/core/math/Offset.scala

```diff
@@ -51,6 +51,16 @@ final case class Offset(p: Offset.Component[Axis.P], q: Offset.Component[Axis.Q]
   def rotate(θ: Angle): Offset =
     Offset.rotateBy(θ)(this)
 
+  /**
+   * Angular separation between two offset positions.
+   */
+  def distance(o: Offset): Angle = {
```

Comment:
I guess this is for relatively small offsets?

Reply:
In practice yes, but it's angular separation not RA/Dec coordinates so the declination is not relevant.  It should work regardless of the separation.

Reply:
ah ok cool!